src/App.js

```diff
@@ -1,13 +1,60 @@
 import React from 'react';
+import { GoodsList } from './components/GoodsList/GoodsList';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRed } from './api/goods';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    rightGoods: [],
+  }
+
+  getGoods = (filter) => {
+    filter()
+      .then((goodsFromServer) => {
+        this.setState({
+          rightGoods: [...goodsFromServer],
```

Comment:
no need to copy array

```suggestion
          rightGoods: goodsFromServer,
```